[fix #2314] do not try to update wallet and transactions when offline

yenda opened the pull request:
This is a quickfix that prevents http calls to update wallet and transactions when device is not online

Fix https://github.com/status-im/status-react/issues/2314

see #1785 for fix of the root problem (lack of timeout in fetch api)

### Steps to test:
- Open Status
- Login on a account witht transactions
- Go in airplane mode
- Open transaction details

yenda committed:
[fix #2314] do not try to update wallet and transactions when offline

yenda commented:
I tested it on Android and it seems to work, however we might want to consider showing the offline warning in the wallet too then so that the user is aware that it's not up-to date ?

asemiankevich commented:
Branch: PR-2945
Android: https://i.diawi.com/faqSde
iOS: https://i.diawi.com/WJVhL3
 
need to rebase this ^

yenda committed:
[fix #] pass network-status to account app-db

when login offline the offline gray box was not shown in chat-list
and the fix in #2314 would not work in that case

asemiankevich commented:
Branch: PR-2945
Android: https://i.diawi.com/gMPQKc
iOS: https://i.diawi.com/6z6169

![image](https://user-images.githubusercontent.com/28752642/34680135-32698f2a-f4a9-11e7-917f-793d971c28ea.png)

STR:

Open Status
Login on a account witht transactions
Go in airplane mode
Open transaction details

this is maybe a regression caused by https://github.com/status-im/status-react/pull/2928. logged the following issue which blocks checking this one https://github.com/status-im/status-react/issues/2957

rasom commented on src/status_im/ui/screens/wallet/events.cljs:
```diff
@@ -86,37 +86,39 @@
-                      (assoc-in [:wallet :balance-loading?] true)
-                      (assoc :prices-loading? true))}))
+  (fn [{{:keys [web3 accounts/current-account-id network network-status] :as db} :db} [_ symbols]]
+    (when-not (= network-status :offline)
```
not sure if this recommendation is applicable here but https://github.com/bbatsov/clojure-style-guide#not-equal

yenda committed:
use not=

asemiankevich commented:
Branch: PR-2945
Android: https://i.diawi.com/xRiDyS
iOS: https://i.diawi.com/yxA5oL

Android works fine in indicated build:
1. can see txn details online
2. can open app (with wifi) then go offline and see txn details
3. can open app (no internet) then go to txn history - no history shown . assume it is the fix - do not fetch the txn list when offline

I am blocked by https://github.com/status-im/status-react/issues/2903 to check it on iO but i confirmed with @yenda that it is not platform specific so lets merge
